compressor/tests/test_offline.py

```diff
@@ -471,6 +460,40 @@ class OfflineCompressTestCaseWithContextGeneratorSuper(
     engines = ('django',)
 
 
+class OfflineCompressStaticUrlIndependenceTestCase(
+        OfflineCompressTestCaseWithContextGenerator):
+    """
+    Test that the offline manifest is independent of STATIC_URL.
+    I.e. users can use the manifest with any other STATIC_URL in the future.
+
+    We use COMPRESS_OFFLINE_CONTEXT generator to make sure that
+    STATIC_URL is not cached when rendering the template.
+    """
+    templates_dir = 'test_static_url_independence'
+    expected_hash = '44ed960a3d1d'
+    additional_test_settings = {
+        'STATIC_URL': '/custom/static/url/',
+        'COMPRESS_OFFLINE_CONTEXT': (
+            'compressor.tests.test_offline.static_url_context_generator'
```

Comment:
did you want to specifically use the offline context thing? if not, you could just use the static tag in the template and wouldn't need the offline context